src/components/DimensionsPanel/DndDimensionsPanel.js

```diff
@@ -19,15 +19,20 @@ export class DndDimensionsPanel extends Component {
     render() {
         return (
             <div className={styles.container} data-test="dimensions-panel">
-                <DimensionFilter
-                    placeholder={i18n.t('Filter dimensions')}
-                    text={this.state.filterText}
-                    onChange={this.onFilterTextChange}
-                    onClear={this.onClearFilter}
-                    disableUnderline={true}
-                    type="search"
-                    dataTest="dimensions-panel-filter"
-                />
+                <div
+                    className={styles.filter}
+                    data-test="dimensions-panel-filter"
```

Comment:
```suggestion
```
`dimensions-panel-filter` is already passed to `DimensionFilter` below so this shouldn't be set on the `div` as well.

Reply:
For reference, `dataTest` / `data-test` is only there for our Cypress tests to find elements